### DISCUSSION TITLE
Вложенность модулей и время выполнения

### DISCUSSION THREAD

**Discussion by Bondian** [Chimildic/goofy#211] [Принцип работы]
Пример двух вариантов записи при работе с кешем. Есть ли какая-то экономия по времени выполнения при больших количествах записываемых файлов в одной функции, или это просто дело стиля и удобства понимания?

```
tracks = Selector.sliceFirst(tracks, 80000);
Cache.write('cachedTracks.json', tracks);
```

  `Cache.write('cachedTracks.json', Selector.sliceFirst(tracks, 80000));`

**Comment by Chimildic** [comment]
Разница между предложенными вариантами нулевая

**Reply by Bondian** [reply]
Спасибо.
